TimePad/Controllers/Details/DetailsViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DetailsViewController.swift
+//  TimePad
+//
+//  Created by Emre Alpago on 10.09.2022.
+//
+
+import UIKit
+
+class DetailsViewController: UIViewController {
+    
+    @IBOutlet private var projectNameLabel: UILabel?
```

Comment:
bunların özellikle mi weak ve ! kısımlarını sildin :D

Reply:
bu konuya bakabilirsin.
ben bu şekilde kullanıyorum

```
@IBOutlet private weak var firstCategoryLabel: UILabel!
```

sthoryboard'dan initialize edildiği için forcelamın makul olduğunu düşünüyorum burada
https://www.reddit.com/r/swift/comments/o2p2np/do_you_usually_make_your_outlet_weak_or_strong/